### PR TITLE
Debugger RegisterWidget Add option to change column's display type

### PR DESCRIPTION
**Pull request by TryTwo**
Adds new context options to change entire column's display type at once.  I simply named it "All Hexadecimal" etc.

### CODE REVIEW

File: Source/Core/DolphinQt/Debugger/RegisterWidget.cpp [hunk 1]
```diff
@@ -221,6 +243,61 @@ void RegisterWidget::ShowContextMenu()
...
 
+    connect(view_hex_column, &QAction::triggered, [this, item] {
+      m_updating = true;
+      auto col = m_table->currentItem()->column();
+      for (int i = 0; i < 32; i++)
+      {
+        auto* update_item = static_cast<RegisterColumn*>(m_table->item(i, col));
+        update_item->SetDisplay(RegisterDisplay::Hex);
+      }
+      m_updating = false;
```
Comment (AdmiralCurtiss) [inline Source/Core/DolphinQt/Debugger/RegisterWidget.cpp:254]: This code is practically identical across the 5 connects, put it into a function or something.

Comment (TryTwo) [reply Source/Core/DolphinQt/Debugger/RegisterWidget.cpp:254]: I was thinking about that, but how do I tell it which RegisterDisplay to use?  Or do I have 5 connects to determine the RegisterDisplay and then one for loop?

**Comment by TryTwo** [comment]
What about this? If yes, should I do it for the individual cell display context options as well?

**Comment by AdmiralCurtiss** [comment]
Yeah, this works. Squash please, and no you don't need to do it for anything else. (But if you do please make it a separate commit.)

**Comment by AdmiralCurtiss** [comment]
@dolphin-emu-bot rebuild

**Comment by AdmiralCurtiss** [comment]
Hm actually testing this, I think it would make more sense for the float registers to set all 64 values instead of just one of the two columns. Can you add that?

**Comment by TryTwo** [comment]
The second column is for paired singles.  When it's displaying the same information as the first column, it's a little helpful having them as different types.

LFS  sets both columns to the same value, LFD sets only the first column, it could be nice if it set both, but not sure if that'd be correct.  PSQ sets the two columns to different values.  I may be missing something, I haven't used it recently.

**Pull request merged**